app/controllers/api/v1/short_form_controller.rb

```diff
@@ -6,6 +6,14 @@ def submit
     logger.debug "application_api_params: #{application_api_params}"
     short_form_validator = ShortFormValidator.new(application_api_params)
     if short_form_validator.valid?
+      # If lease information is available, submit it separately.
+      if application_api_params.key?(:lease)
+        response = lease_service.submit_lease(application_api_params[:lease],
```

Comment:
what do we get in the response here? should be return this in the json response the same way we do it for the application?

Reply:
If it's a create, we get the lease id or False, if it's an update we get True or False

Reply:
I see ok. then no changes here.